aws/table_aws_cloudfront_response_headers_policy.go

```diff
@@ -0,0 +1,212 @@
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getETagValue,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "response_headers_policy_config",
+				Description: "A response headers policy contains information about a set of HTTP response headers and their values. CloudFront adds the headers in the policy to HTTP responses that it sends for requests that match a cache behavior that’s associated with the policy.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ResponseHeadersPolicy.ResponseHeadersPolicyConfig"),
+			},
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ResponseHeadersPolicy.ResponseHeadersPolicyConfig.Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAccountARN,
+				Transform:   transform.FromValue().Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listCloudFrontResponseHeadersPolicy(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("tableAwsCloudFrontFunction.listCloudFrontResponseHeadersPolicy")
+
+	svc, err := CloudFrontService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	input := cloudfront.ListResponseHeadersPoliciesInput{
+		MaxItems: aws.Int64(100),
+	}
+
+	// Reduce the basic request limit down if the user has only requested a small number of rows
+	limit := d.QueryContext.Limit
+	if d.QueryContext.Limit != nil {
+		if *limit < *input.MaxItems {
+			if *limit < 1 {
+				input.MaxItems = aws.Int64(1)
+			} else {
+				input.MaxItems = limit
+			}
+		}
+	}
+
+	// Additonal Filter
+	policyTypeColumn := d.KeyColumnQuals["type"]
+	if policyTypeColumn != nil {
+		input.Type = aws.String(policyTypeColumn.GetStringValue())
+	}
+
+	pagesLeft := true
+	for pagesLeft {
+		data, err := svc.ListResponseHeadersPolicies(&input)
+		if err != nil {
+			plugin.Logger(ctx).Error("ListResponseHeadersPolicies", "ERROR", err)
+			return nil, err
+		}
+
+		for _, policy := range data.ResponseHeadersPolicyList.Items {
+			d.StreamListItem(ctx, policy)
+		}
+
+		if data.ResponseHeadersPolicyList.NextMarker != nil {
+			pagesLeft = true
+			input.Marker = data.ResponseHeadersPolicyList.NextMarker
+		} else {
+			pagesLeft = false
+		}
+	}
+
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+func getETagValue(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("tableAwsCloudFrontFunction.getCloudFrontResponseHeadersPolicy")
+
+	var id string
+
+	if h.Item != nil {
+		item := h.Item.(*cloudfront.ResponseHeadersPolicySummary)
+		id = *item.ResponseHeadersPolicy.Id
+	} else {
+		id = d.KeyColumnQuals["id"].GetStringValue()
+	}
+
+	if id == "" {
```

Comment:
This is a separate hydrate call for a single column. We will not get `id` from key columns here. Please make the changes.